terratest/src/test/pega/pega-tier-deployment-artifactory-certificates_test.go

```diff
@@ -27,6 +27,7 @@ func TestPegaDeploymentWithArtifactoryCerts(t *testing.T) {
 					"global.provider":               vendor,
 					"global.actions.execute":        operation,
 					"installer.upgrade.upgradeType": "zero-downtime",
+					"global.storageClassName": "storage-class",
```

Comment:
This change is not required in all the test cases. In fact, create another test case in pega-tier-deployment_test.go with storage-class and perform the required assertions. All others test need to not be touched